src/matchers/toHaveBeenCalledBefore/index.js

```diff
@@ -18,8 +20,24 @@ const failMessage = (firstInvocationCallOrder, secondInvocationCallOrder) => ()
   'Received second mock with invocationCallOrder:\n' +
   `  ${printReceived(secondInvocationCallOrder)}`;
 
+const mockCheckFailMessage = (value, isFirstValue) => () =>
+  matcherHint('.toHaveBeenCalledAfter') +
+  '\n\n' +
+  `Matcher error: ${isFirstValue ? 'first' : 'second'} value must be a mock or spy function\n` +
```

Comment:
im not sure on `first` and `second` being the right wording here, could we potentially cause ambiguation for users if they read `second value must be a mock or spy function` and presume this means they have to give a second value to the matcher, not that the matchers first value isn't a mock/spy?

Reply:
Yes, I also spent some time figuring out how to word it better. Probably `actual` and `expected` instead of `first` and `second`?

Reply:
Replaced with "Received" and "Expected" with red and green color, just as in matcher hint.